workspaces/marketplace/plugins/marketplace-common/src/types.ts

```diff
@@ -84,6 +92,36 @@ export interface MarketplacePluginSpec extends JsonObject {
     appconfig?: string;
   };
 }
+
+/**
+ * @public
+ */
+export interface MarketplacePackageSpec extends JsonObject {
+  packageName: string;
+  dynamicArtifact?: string;
+  author?: string;
+  support?: string;
+  lifecycle?: string;
+  backstage?: MarketplacePackageBackstage;
+  appConfigExample?: AppConfigExample;
+}
+
+/**
+ * @public
+ */
+export interface AppConfigExample extends JsonObject {
+  title: string;
+  content: string;
```

Comment:
I think we could allow here also a YAML => Json structure

```suggestion
  content: string | JsonObject;
```